packages/devtools_app/lib/src/screens/memory/shared/heap/class_filter.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../primitives/class_name.dart';
+
+enum ClassFilterType {
+  all,
```

Comment:
can this be 'showAll', or 'none'? 'all' by itself makes it seem like we are filtering all classes instead of showing all (i.e. no filter)